Modules/Filtering/ImageGradient/test/itkVectorGradientMagnitudeImageFilterTest1.cxx

```diff
@@ -27,7 +27,7 @@
 int
 itkVectorGradientMagnitudeImageFilterTest1(int argc, char * argv[])
 {
-  if (argc != 7)
+  if (argc < 7)
   {
     std::cerr << "Missing Parameters." << std::endl;
```

Comment:
I don't know the answer to this question ... is there an ITK style for using `argc != 7` vs. `argc < 7`.  Since I don't know any reason why `argc > 7` would be desirable in this case, I'd think that an error would be appropriate in that case too.

Reply:
Thanks Lee. Changed back to `!=` in 79c65d7. You're right as per the ITK SW Guide (https://itk.org/ItkSoftwareGuide.pdf; section C.24, p. 370: C.24). It might have been myself that wrote that part, but at times I am not consistent myself wither. Sorry.

I do not mean to change all tests in this PR to that match convention as I am not meaning to be exhaustive. That can be done in a separate PR.